go.mod

```diff
@@ -31,3 +31,5 @@ require (
 // Fix for `[` in help messages and shell completion code
 // See https://github.com/spf13/cobra/pull/899
 replace github.com/spf13/cobra => github.com/chmouel/cobra v0.0.0-20191021105835-a78788917390
+
+go 1.13
```

Comment:
The results are different for codegen if ran with go 1.13 and go1.12.x, CI is running go1.12.
Saw this issue in other PRs as well.

Reply:
Thanks, you saved me a good amount of frustration. :)